docs/api/blockchain/hotspots.mdx

```diff
@@ -1968,10 +1968,7 @@ _Query Parameters_
 GET https://api.helium.io/v1/hotspots/:address/witnesses
 ```
 
-Retrieves the last known list of witnesses for a given hotspot.In addition to
-the hotspot result information, the result will also include a `witness_for` and
-a `witness_info` field which are the given hotspot address and bucket histogram
-information on witnessed signal strengths.
+Retrieves the last known list of witnesses for a given hotspot.
```

Comment:
It's actually a list of witnesses for a given hotspot over a number of  blocks

Reply:
@roberthenderson If we can get the language updated to what marc suggested above, we're good to go. Thanks!

Reply:
@PharkMillups was that meant for me or actually roberthenderson? Also, what is that number of blocks?

Reply:
Done.